modules/op-release-notes-1-11.adoc

```diff
@@ -0,0 +1,194 @@
+// Module included in the following assembly:
+//
+// * cicd/pipelines/op-release-notes.adoc
+:_content-type: REFERENCE
+[id="op-release-notes-1-11_{context}"]
+= Release notes for {pipelines-title} General Availability 1.11
+
+With this update, {pipelines-title} General Availability (GA) 1.11 is available on {product-title} 4.12 and later versions.
+
+[id="new-features-1-11_{context}"]
+== New features
+
+In addition to fixes and stability improvements, the following sections highlight what is new in {pipelines-title} 1.11.
+
+[id="pipelines-new-features-1-11_{context}"]
+=== Pipelines
+
+* With this update, you can use {pipelines-title} on the {product-title} cluster that runs on ARM hardware. This update includes support for the `ClusterTask` resources where images are available and the Tekton CLI tool on ARM hardware.
+
+* This update adds support for results, object parameters, array results, and indexing into an array when you set the `enable-api-fields` feature flag to `beta` in the `TektonConfig` CR.
+
+* With this update, propagated parameters are now part of a stable feature. This feature enables interpolating parameters in embedded specifications to reduce verbosity in Tekton resources. 
+
+* With this update, propagated workspaces are now part of a stable feature. You can enable the propagated workspaces feature by setting the `enable-api-fields` feature flag to `alpha` or `beta`.
+
+* With this update, the `TaskRun` object fetches and displays the init container failure message to users when a pod fails to run.
+
+* With this update, you can replace parameters, results, and the context of a pipeline task while configuring a matrix:
+** Replace an array with an array parameter or a string with a string, array, or object parameter in the `matrix.params` configuration.
+** Replace a string with a string, array, or object parameter in the `matrix.include` configuration.
+** Replace the context of a pipeline task with another context in the `matrix.include` configuration.
+
+* With this update, the `TaskRun` validation process also validates the `matrix.include` parameters. The validation checks whether all parameters have values and match the specified type, and object parameters have all the keys required.
+
+* This update adds a new field named `default-resolver-type` in the `default-configs` config map. You can set the value of this field to configure a default resolver.
+
+* With this update, you can define and use a `PipelineRun` context variable in the `pipelineRun.workspaces.subPath` configuration.
+
+* With this update, the `ClusterResolver`, `BundleResolver`, `HubResolver`, and `GitResolver` features are now available by default. 
++
+[NOTE]
+====
+The `taskref.bundle` and `pipelineref.bundle` bundles are deprecated and will be removed in a future release.
+====
```

Comment:
Should this not be moved to the Deprecated Features section? Why is it appearing here?

Reply:
Thank you for pointing this. Moved to the deprecated section.